anchore-k8s-inventory.yaml

```diff
@@ -14,6 +14,17 @@ log:
 # enable/disable checking for application updates on startup
 check-for-app-update: true
 
+registration:
```

Comment:
Yes, that's a good point. I'll change that.

Overall, the I'm mulling over how to best describe and explain the meaning and purpose of the registration-id.
As long as the deployer uses our helm charts and deploys the agent that way, it is best to just leave the registration-id unspecified. It will then be the set to the uid of the K8s Deployment.